python/cugraph/cugraph/tests/layout/test_force_atlas2.py

```diff
@@ -13,13 +13,48 @@
 
 import time
 import pytest
-import scipy.io
-from sklearn.manifold import trustworthiness
 
-import cudf
 import cugraph
+from cugraph.structure import number_map
 from cugraph.internals import GraphBasedDimRedCallback
-from cugraph.datasets import karate, polbooks, dolphins, netscience
+from sklearn.manifold import trustworthiness
+import scipy.io
+from cugraph.datasets import (
+    karate,
+    polbooks,
+    dolphins,
+    netscience,
+    dining_prefs,
+)
+
+# Temporarily suppress warnings till networkX fixes deprecation warnings
+# (Using or importing the ABCs from 'collections' instead of from
+# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
+# python 3.7.  Also, these import fa2 and import networkx need to be
+# relocated in the third-party group once this gets fixed.
```

Comment:
This comment should be removed; the code to suppress warnings isn't here anymore.